Makefile

```diff
@@ -1,3 +1,7 @@
+# Makefile
+
+export LC_ALL ?= C.UTF-8
```

Comment:
I'm not sure about the "?=", this still takes LC_ALL from the user environment if it is set.

Reply:
In the issue it was ``LC_ALL=``. We can also always set LC_ALL=C.UTF-8. Should be also fine.